paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1028,6 +1028,25 @@ public class CoreOptions implements Serializable {
                             "Parameter string for the constructor of class #. "
                                     + "Callback class should parse the parameter by itself.");
 
+    public static final ConfigOption<String> PARTITION_MARK_DONE_ACTION =
```

Comment:
Regenerate doc?

Reply:
OK，would move it to core_configuration.html